portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+    public Comment() {
```

Comment:
This constructor and the other one should be package-private

Reply:
So the builder in the same package can access them but other packages cannot

Reply:
If I remove the public keyword then it will be package visible by default right?

Reply:
Yes